### PR TITLE
Add grade and publish type in books

### PR DESCRIPTION
**Pull request by bimal125**
## Changes

* Add grade choice field
* Add publish type choice field

Mention related users here if any.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: apps/book/models.py [hunk 1]
```diff
@@ -62,6 +62,24 @@ class LanguageType(models.TextChoices):
...
+        GRADE_SIX = 'grade_six', 'Grade six'
+        GRADE_SEVEN = 'grade_seven', 'Grade seven'
+        GRADE_EIGHT = 'grade_eight', 'Grade eight'
+        GRADE_NINE = 'grade_nine', 'Grade nine'
+        GRADE_TEN = 'grade_ten', 'Grade ten'
+        GRADE_ELEVEN = 'grade_eleven', 'Grade eleven'
+        GRADE_TWELVE = 'grade_twelve', 'Grade twelve'
+
+    class PublishType(models.TextChoices):
+        PUBLISH = 'publish', 'Publish'
```
Comment (thenav56) [inline apps/book/models.py]: If it's just 2 states, then maybe use is_draft or is_published? cc: @frozenhelium @tnagorra

File: apps/book/models.py [hunk 2]
```diff
@@ -62,6 +62,24 @@ class LanguageType(models.TextChoices):
...
+        GRADE_THREE = 'grade_three', 'Grade three'
+        GRADE_FOUR = 'grade_four', 'Grade four'
+        GRADE_FIVE = 'grade_five', 'Grade five'
+        GRADE_SIX = 'grade_six', 'Grade six'
+        GRADE_SEVEN = 'grade_seven', 'Grade seven'
+        GRADE_EIGHT = 'grade_eight', 'Grade eight'
+        GRADE_NINE = 'grade_nine', 'Grade nine'
+        GRADE_TEN = 'grade_ten', 'Grade ten'
+        GRADE_ELEVEN = 'grade_eleven', 'Grade eleven'
+        GRADE_TWELVE = 'grade_twelve', 'Grade twelve'
```
Comment (thenav56) [inline apps/book/models.py]: Looks like we need only 3 grades for now. Let's do this.
```suggestion
        GRADE_1 = 'grade_1', _('Grade 1')
        GRADE_2 = 'grade_2', _('Grade 2')
        GRADE_3 = 'grade_3', _('Grade 3')
```

File: apps/book/models.py (hunk 1 above)
Comment (thenav56) [reply apps/book/models.py]: After talking with @amarshahi, looks like just **is_draft** should be okay for now.

File: apps/book/models.py (hunk 2 above)
Comment (bimal125) [reply apps/book/models.py]: @thenav56  Verbose name changed

File: apps/book/models.py (hunk 1 above)
Comment (bimal125) [reply apps/book/models.py]: @thenav56  Change to boolean field is_published

File: apps/book/models.py (hunk 2 above)
Comment (thenav56) [reply apps/book/models.py]: @bimal125 Please re-read the message above.

**Review by thenav56 — APPROVED** [review]

**Review by tnagorra — APPROVED** [review]

**Pull request merged**